### PR TITLE
Add framework application actions which returns all registered actions and events - Closes #6217

### LINKED ISSUE #6217: Add framework application actions which returns all registered actions and events
### Description

Add application actions
`app:getRegisteredActions` and `app:getRegisteredEvents` where each returns `string[]` with all registered actions and events

### Motivation

In order to show all the possible actions to invoke or events to subscribe, framework needs to return the list.

### Acceptance Criteria

- Add functional test to see if all registered actions and events are returned including the one defined in modules and plugins

### Additional Information
Related Issue #6216

### PR DESCRIPTION
**Pull request by Incede**
### What was the problem?

This PR resolves #6217 

### How was it solved?

* Create actions to return all registered actions & events

### How was it tested?
* Added functional test

### CODE REVIEW

File: framework/src/application.ts [hunk 1]
```diff
@@ -447,6 +447,12 @@ export class Application {
 				getNodeInfo: {
 					handler: () => this._node.actions.getNodeInfo(),
 				},
+				getRegisteredActions: {
+					handler: () => this._controller.bus.getActions(), // this._getRegisteredActions(),
```
Comment (shuse2) [inline framework/src/application.ts:451]: what is this comment?

**Review by shuse2 — APPROVED** [review]

File: framework/src/application.ts (hunk 1 above)
Comment (Incede) [reply framework/src/application.ts]: typo, removed!

File: framework/test/functional/rpc-api/ipc/ipc_client_in_memory.spec.ts [hunk 2]
```diff
@@ -103,6 +103,14 @@ describe('api client ipc mode', () => {
 				client.invoke('app:getAccount', { address: 'randomString*&&^%^' }),
 			).rejects.toThrow('Specified key accounts:address: does not exist');
 		});
+
+		it('should return a list of registered actions', async () => {
+			const actions = await await client.invoke('app:getRegisteredActions');
+			expect(actions).toBeArray();
+			expect(actions).toContain('app:getConnectedPeers');
+			expect(actions).toContain('dpos:getAllDelegates');
+			expect(actions).toContain('hello:callGreet');
```
Comment (nazarhussain) [inline framework/test/functional/rpc-api/ipc/ipc_client_in_memory.spec.ts:112]: Either we match exact array or match the length of array as well to be sure there are only these actions available.

File: framework/test/functional/rpc-api/ipc/ipc_client_in_memory.spec.ts [hunk 3]
```diff
@@ -115,6 +123,13 @@ describe('api client ipc mode', () => {
 			expect(newBlockEvent.length).toBeGreaterThan(0);
 			expect(newBlockEvent[0]).toHaveProperty('block');
 		});
+		it('should return a list of registered events', async () => {
+			const events = await await client.invoke('app:getRegisteredEvents');
+			expect(events).toBeArray();
+			expect(events).toContain('app:ready');
+			expect(events).toContain('token:registeredToBus');
+			expect(events).toContain('hello:greet');
```
Comment (nazarhussain) [inline framework/test/functional/rpc-api/ipc/ipc_client_in_memory.spec.ts:131]: Same here.

**Review by nazarhussain — CHANGES_REQUESTED** [review]

**Review by nazarhussain — APPROVED** [review]

**Pull request merged**